### ISSUE TITLE
Docker build sometimes fails during installation step

### ISSUE THREAD

**Issue by aileenzeng** [jonfroehlich/makeabilitylabwebsite#365]
![image](https://user-images.githubusercontent.com/25534091/40739141-366ce926-63fa-11e8-8150-465e0210c772.png)

This seems to be a problem with a missing argument from wand. (This also doesn't happen all the time)

**Comment by jonfroehlich** [comment]
Thanks @aileenzeng. To make sure our Issues are accessible to all Makeability Lab members, can you try to copy/paste log files directly and use markdown to wrap them in ```?

```
This is an example
````

**Comment by aileenzeng** [comment]
@jonfroehlich Oops - sorry about that! Here's the log:

```
Step 8/12 : RUN pip install -r requirements.txt
 ---> Running in a8ed0aeb433a
Collecting django==1.9 (from -r requirements.txt (line 3))
  Downloading https://files.pythonhosted.org/packages/ea/9b/b5a6360b3dfcd88d4bad70f59da26cbde4bdec395a31bb26dc840e806a50/Django-1.9-py2.py3-none-any.whl (6.6MB)
Collecting wand==0.4.2 (from -r requirements.txt (line 4))
  Downloading https://files.pythonhosted.org/packages/60/14/5bb6919170f796bcc0b3b42c41a8679394e397468f22e5d89159bcd39197/Wand-0.4.2.tar.gz (63kB)
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-duvy85go/wand/setup.py", line 8, in <module>
        from wand.version import VERSION
      File "/tmp/pip-install-duvy85go/wand/wand/version.py", line 110, in <module>
        *map(int, MAGICK_RELEASE_DATE_STRING.split('-')))
    TypeError: Required argument 'month' (pos 2) not found
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-install-duvy85go/wand/
The command '/bin/sh -c pip install -r requirements.txt' returned a non-zero code: 1
```

It seems like this problem is actually a bug with wand itself. Somebody reported the bug [here](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=841548) -- looks like it got fixed in 0.4.4

**Comment by aileenzeng** [comment]
Updated line 4 in `requirements.txt`:

Original:
```
wand==0.4.2
```
Current:
```
wand==0.4.4
```

Changes made in commit `7654120b62af95d904ba9529b539193ced8ca3e9`
